programs/survival/bucketstack.sc

```diff
@@ -0,0 +1,179 @@
+__config() ->
+(
+	//change these number to whatever you want the maximum stack size to be
+	//note that the max scarpet can inventory_set() is 2^31 - 1
+	global_buckets = m(
+		l('water',1),
+		l('lava',1),
+		l('milk',1),
+		l('pufferfish',1),
+		l('salmon',1),
+		l('cod',1),
+		l('tropical_fish',1),
+		l('axolotl',1),
+		l('powder_snow',1)
+	);

```

Comment:
I think that an ingame way to change this config would be really cool
`/bucketstack max <item> set <count>`
`/bucketstack max <item> get`

Reply:
Done